stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/transactions.py

```diff
@@ -104,7 +103,7 @@ def create_collection(self, model: stac_types.Collection, **kwargs):
     def update_item(self, model: stac_types.Item, **kwargs):
         """Update item."""
         base_url = str(kwargs["request"].base_url)
-        now = datetime.utcnow().strftime(DATETIME_RFC339)
```

Comment:
I'm removing all the uses of DATETIME_RFC339 is stac-fastapi right now, and we can remove all the uses of it in this repo after that's done.